### DISCUSSION TITLE
Companies & Websites Using Biome

### DISCUSSION THREAD

**Discussion by ematipico** [biomejs/biome#1582] [General]
If you are using Biome at your company or organization, please share the URL here! Share with us a logo that you would like to feature on the webpage. 

We will feature you on our upcoming `/users` page.

**Comment by marcusbuffett** [comment]
We’ve been using it for months for https://chessbook.com , and loving it!
![IMG_2813](https://github.com/biomejs/biome/assets/1834328/a3b95d90-8128-4154-9a88-916cfc1ed6f2)

**Reply by Conaclos** [reply]
Have you a SVG logo?

**Comment by chenjiahan** [comment]
Rsbuild and Rspress use Biome for linting and formatting:

- https://github.com/web-infra-dev/rsbuild
- https://github.com/web-infra-dev/rspress

<img src="https://github.com/biomejs/biome/assets/7237365/c63b4b8c-b7a7-4029-975e-35da792111e2" width="320" />

**Reply by Conaclos** [reply]
SVG format: https://github.com/web-infra-dev/rspack/discussions/4845

**Comment by XiNiHa** [comment]
At PortOne, we're using it to format and lint a React-based admin console application.
Although we prefer the text logo to be used, any logo that matches well with the overall style is okay.
Let me know if any adjustments on the logo are needed!
![portone-square](https://github.com/biomejs/biome/assets/8155259/131d88a3-046e-4ed5-bbae-4d5851816030)
![portone-textual](https://github.com/biomejs/biome/assets/8155259/d81025d6-ebdf-48ca-bb6b-12d5bd8b33d7)
![portone-textual-wh](https://github.com/biomejs/biome/assets/8155259/dfe0c274-abd5-4113-a4c4-b01b3cbff93d)

**Reply by disjukr** [reply]
https://portone.io

**Reply by Conaclos** [reply]
Have you a SVG logo?

**Reply by XiNiHa** [reply]
Sadly our logo includes an angular gradient, which cannot be exported as SVG ;(

**Reply by XiNiHa** [reply]
(the two text logo variants are already SVG)

**Reply by disjukr** [reply]
how about this: https://portone-logo-to-svg.vercel.app/

**Comment by itsMapleLeaf** [comment]
https://astro.build/ 👉 😏 👉 

(not on the website main itself yet, but on a handful of other repos)

**Comment by thejackshelton** [comment]
The `@qwikdev/astro` integration!

https://github.com/QwikDev/astro

**Comment by afonsojramos** [comment]
https://www.gitify.io/

We've just started using it ☝️

**Comment by mrazauskas** [comment]
TSTyche is using Biome. Really happy about it!

https://tstyche.org

![tstyche](https://github.com/biomejs/biome/assets/72159681/7edde79c-7f1d-4242-a421-c4810b032bed)

**Comment by JamesReich** [comment]
We're using Biome at [https://www.cloak.chat](url)

![full-logo-white](https://github.com/biomejs/biome/assets/22831525/399c3633-a425-4dbe-ab05-949d0cbcfdd6)

**Comment by SaadBazaz** [comment]
We're using Biome at https://grayhat.studio

![Grayhat Developers PVT LTD Full Logo ](https://github.com/user-attachments/assets/580cccac-4f39-4c89-baa3-bd33c2e398bf)

**Comment by PaulRBerg** [comment]
Using Biome at Sablier: https://sablier.com

![Sablier Logo](https://raw.githubusercontent.com/sablier-labs/branding/6f3e5cdefe6d0ded9f9de99348e7139e391bf330/logo/wide/png/logo-wide-gradient-dark.png)

**Comment by gpbl** [comment]
We replaced eslint with Biome in https://daypicker.dev - hoping [markdown](https://github.com/biomejs/biome/issues/3718) and [jsdocs](https://github.com/biomejs/biome/discussions/741) will be supported soon 🙏

**Comment by jpike88** [comment]
![logo](https://github.com/user-attachments/assets/19088919-480b-4d3a-8157-5b01d1199b2f)

Aquipa
